retriever/__main__.py

```diff
@@ -238,6 +238,18 @@ def main():
             for dataset in scripts:
                 print("=> Installing", dataset.name)
                 try:
+                    if engine.name == "HDF5":
```

Comment:
Note: Move this to the interface

Reply:
I feel like we can remove this whole  block and use the functions that are provided by the interface 

```python
        if scripts:
            if args.dataset.endswith('.zip') or (hasattr(args, 'hash_value') and
                                                 args.hash_value):
                _install(vars(args), debug=debug, use_cache=use_cache)
                return
            for dataset in scripts:
                print("=> Installing", dataset.name)
                try:
                    if engine.name == "HDF5":
                        sqlite_opts = {
                            'command': 'install',
                            'dataset': dataset,
                            'engine': 'sqlite',
                            'file': (args.file.split("."))[0] + ".db",
                            'table_name': args.table_name,
                            'data_dir': args.data_dir
                        }
                        sqlite_engine = choose_engine(sqlite_opts)
                        dataset.download(sqlite_engine, debug=debug)
                        dataset.engine.final_cleanup()
                    dataset.download(engine, debug=debug)
                    dataset.engine.final_cleanup()
                except KeyboardInterrupt:
                    pass
                except Exception as e:
                    print(e)
                    if debug:
                        raise
            print("Done!")

```

to something like 

```python
        if scripts:
            _install(vars(args), debug=debug, use_cache=use_cache)
```
what do you think @apoorvaeternity